signbank/dictionary/csv_import.py

```diff
@@ -721,3 +721,157 @@ def confirm_import_qualtrics_csv(request):
             "missing_gloss_question_pairs": missing_gloss_pk_question_pairs
         }
     )
+
+
+@login_required
+@permission_required("dictionary.import_csv")
+def import_manual_validation(request):
+    """
+    Import ManualValidationAggregations from a CSV file
+    """
+    # Make sure that the session variables are flushed before using this view.
+    request.session.pop("group_row_map", None)
+    request.session.pop("glosses", None)
+
+    if not request.method == "POST":
+        # If request type is not POST, return to the original form.
+        csv_form = CSVFileOnlyUpload()
+        return render(request, "dictionary/import_manual_validation_csv.html",
+                      {"import_csv_form": csv_form}, )
+
+    form = CSVFileOnlyUpload(request.POST, request.FILES)
+
+    if not form.is_valid():
+        # If form is not valid, set a error message and return to the original form.
+        messages.add_message(request, messages.ERROR,
+                             _("The provided CSV-file does not meet the requirements "
+                               "or there is some other problem."))
+        return render(request, "dictionary/import_manual_validation_csv.html",
+                      {"import_csv_form": form}, )
+
+    group_row_map = {}
+    group_gloss_count = {}
+    glosses = []
+    required_headers = [
+        "group",
+        "idgloss",
+        "yes",
+        "no",
+        "abstain or not sure",
+        "comments"
+    ]
+    try:
+        validation_record_reader = csv.DictReader(
+            codecs.iterdecode(form.cleaned_data["file"], "utf-8"),
+            delimiter=",",
+            quotechar='"'
+        )
+        for header in required_headers:
```

Comment:
**suggestion**: instead of iterating like this, use set difference to find any missing headers:

```python
missing_headers = set(required_headers) - set(validation_record_reader.fieldnames)
```
<img width="396" alt="Screenshot 2024-02-26 at 14 34 03" src="https://github.com/ODNZSL/NZSL-signbank/assets/2138067/4363a4c7-3123-4251-b02a-c4e5c3974452">